utilities/helpers.js

```diff
@@ -5,4 +5,12 @@ export const formatDate = function ( date ) {
     const month = formattedDate.toLocaleString( 'default', { month: 'short' } )
     const year = formattedDate.getFullYear()
     return `${ month } ${ day }, ${ year }`
+}
+
+// format the url of a publisher image so it works with our design system components
+export const publisherImageFormatter = ( url ) => {
```

Comment:
Still not sure why this didn't work... and I had to inline this transform. Maybe an SSR thing?

Reply:
maybe, I'm not sure either!